main.py

```diff
@@ -1,10 +1,59 @@
+import json
+from typing import Union
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        data = json.load(file)
+        for player_nickname, player_data in data.items():
+            race = create_race(player_data)
+            guild = create_guild(player_data)
+            create_skills(player_data)
+
+            if not Player.objects.filter(nickname=player_nickname).exists():
```

Comment:
In this task, there's no need to check if a player already exists. It doesn't hurt, but it's certainly not necessary.